src/client/pydaos/raw/daos_api.py

```diff
@@ -2362,7 +2362,7 @@ def get_function(self, function):
             # For most functions, we need to ensure
             # that daos_init() has been called before
             # invoking anything.
-            self._dc = DaosClient()
+            self._dc = DaosClient(noeq = 1)
```

Comment:
This would then become
```suggestion
            self._dc = DaosClient()
            # pylint: disable=protected-access
            self._dc._eq = False
```